spec/rspec/core/rake_task_spec.rb

```diff
@@ -163,14 +163,14 @@ def silence_output(&block)
       it "removes the environment variables", :if => RUBY_VERSION >= '1.9.0', :unless => RSpec::Support::Ruby.jruby? do
         with_env_vars 'MY_ENV' => 'ABC' do
           if RSpec::Support::OS.windows?
-            essential_shell_variables = /\["ANSICON", "ANSICON_DEF", "HOME", "TMPDIR", "USER"\]/
+            essential_shell_variables = "[\"ANSICON\", \"ANSICON_DEF\", \"HOME\", \"TMPDIR\", \"USER\"]"
           else
-            essential_shell_variables = /\["PWD"(?:, "SHLVL")?(?:, "_")?(?:, "__CF_USER_TEXT_ENCODING")?\]/
```

Comment:
Wasn't this a quoted regexp?

Reply:
@pirj Yes, I thought it would be more readable as a string instead of regexp. I could change it back if desired since this is mostly a style change.

Reply:
Changing this broke the build, the string isn't consistent which is why its a regular expression.

Reply:
A nicer way to do this could be:
```ruby
%w[PWD SHLVL _ __CF_USER_TEXT_ENCODING].sort.select { |var| ENV[var] }.inspect
```

Reply:
The issue is that CF encoding isn't always present, like on travis where its just PWD, please revert to the working example.

Reply:
@JonRowe I've reverted this back to the working example. Thanks.